src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid.EntityFrameworkAdapter/src/Microsoft.AspNetCore.Components.QuickGrid.EntityFrameworkAdapter.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <IsTrimmable>true</IsTrimmable>
+  </PropertyGroup>
```

Comment:
If this project will produce a shipping package, it should probably have `$(Description)` and `$(GenerateDocumentationFile)` properties. Also, check the src/Components/Directory.Build.props file to confirm the `$(PackageTags)` property there is sufficient.

Same for the other new project except it at least has a `$(Description)` property.

If this will produce a transport package, add `<IsShippingPackage>false</IsShippingPackage>` instead. I doubt this should be non-packable (because I don't see anything wrapping the built content) but `<IsPackable>false</IsPackable>` would flip that bit.